cmd/healthcheck/healthcheck.go

```diff
@@ -31,6 +32,7 @@ func healthCheck() error {
 }
 
 func main() {
+	log.SetLevel(log.DebugLevel)
```

Comment:
How would the logger be passed by the service/RP to the plugin e.g. in ACS E: https://github.com/Azure/acs-engine/blob/master/pkg/operations/kubernetesupgrade/upgradecluster.go#L64?